files/build_templates/qos_config.j2

```diff
@@ -68,11 +69,39 @@
 {%- endfor %}
 {%- set port_names_active = port_names_list_active | join(',') -%}
 
+{# Generate uplink and downlink port lists based on router type and neighbors #}
+{%- if DEVICE_METADATA is defined and DEVICE_METADATA['localhost'] is defined %}
```

Comment:
It appears that you use different logic to determine the uplink/downlink port list.
Why not using same logic?

Reply:
Will use this file's qos_config.j2, and remove buffers_config.j2 definitions.